ee/desktop/notify/notify_darwin.go

```diff
@@ -0,0 +1,43 @@
+package notify
+
+/*
+#cgo darwin CFLAGS: -DDARWIN -x objective-c
+#cgo darwin LDFLAGS: -framework Foundation -framework UserNotifications
+
+#include <stdbool.h>
+#include <stdlib.h>
+
+void sendNotification(char *title, char *content);
+*/
+import "C"
+import (
+	"fmt"
+	"os"
+	"strings"
+	"unsafe"
+)
+
+func (d *DesktopNotifier) SendNotification(title, body string) error {
+	if !isBundle() {
```

Comment:
why does it have to be a bundle, is there entitlements or something needed? mind leaving a comment with a brief explanation?

Reply:
Yeah exactly -- it will actually panic if it's not inside a bundle, so it's definitely important to document -- I'll update!